rednotebook/gui/menu.py

```diff
@@ -187,8 +187,17 @@ def select_journal(self, action, title, message):
             return
 
         if action == 'saveas':
+            # TODO: read directory name from self.dirs
```

Comment:
I think it's best to add a property `Filesystem.media_dir(self): return os.path.join(self.data_dir, 'media')`.